docs/reference/cluster_manifest.md

```diff
@@ -97,6 +97,12 @@ Those are parameters grouped directly under  the `spec` key in the manifest.
    is taken from the `pod_priority_class_name` operator parameter, if not set
    then the default priority class is taken. The priority class itself must be defined in advance.
    
+* **shmVolume**
```

Comment:
1. @Jan-M  just to double check - the original issue description talked about modifying the configmap, not only cluster manifests. Can it be the case that we will need to enable this feature for a bunch of PG clusters, and then it will be easier to enable it in the operator and let it take care of the rest  ?  IMHO most clusters will no need this feature, and we can leave it on the level of the cluster manifest as it is currently implemented

2. is the intention of this feature to mount a `tmpfs` volume at `/dev/shm` without restricting the volume's size ?

3. ``shmVolume: true`` is probably not the most descriptive name for that option. Sth like `enableExtraSharedMemory` sounds imo better cause it reflects the purpose of the feature.